src/app/components/source/UserSecurity.js

```diff
@@ -1,4 +1,5 @@
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
Reading these changes over, they look good, but I will make a note on the Jira issue that I strongly recommend we very thoroughly test 2FA flows when this is in QA. It's very complicated and we can't mess it up! @caiosba

Reply:
Already I tested User features related to Security and Privacy locally and added this one to testing list in jira ticket